.pre-commit-config.yaml

```diff
@@ -38,6 +38,7 @@ repos:
     rev: 25.1.0
     hooks:
       - id: black
+        exclude: core/keys/symbols.py$
```

Comment:
After a bit of research, this seems like the least-worst approach to addressing the formatting.

Reply:
Couldn't we achieve this with `# fmt: off` in the file instead?

Reply:
Done.